naturebridge.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: page
+---
+### Venue information
+
+2019 Ruby by the Bay will be held March 28 - April 1 (Friday - Monday) at NatureBridge in Sausalito, California.
+
+![Aerial photo of venue next to the ocean](/images/naturebridge_from_above.jpg "Aerial photo of venue next to the ocean"){:class="img-responsive"}
+
+###### What is this beautiful place??
+
+![Photo of NatureBridge signage in front of some of their buildings](/images/naturebridge_sign.jpg "Photo of NatureBridge signage in front of some of their buildings"){:class="img-responsive"}
+
+NatureBridge is a nonprofit that focuses on educating kids about the environment.  They have school groups during the school year, and run day camps during the summers.
+
+###### Seriously how is it this beautiful??
+
+NatureBridge is lucky enough to lease their space from the National Park Service. It's housed in Golden Gate National Recreation Area. The campus was originally created by the Army.
+
+###### What will sleeping arrangements be like?
+We have four bunkhouses:
+* Non-binary people (Owl's Roost)
```

Comment:
I commented about this in Slack but not sure if anyone saw it.  
I wonder if we will have enough non-binary people who register to fill a dorm?  Maybe we don't need to get so specific yet and we can see who registers and what their needs are and then work to make our space fit the needs of them?